ocfweb/docs/docs/staff/procedures/accounts/renaming.md

```diff
@@ -0,0 +1,100 @@
+[[!meta title="Renaming Accounts"]]
+
+Sometimes OCF accounts need to be renamed. Users sometimes make mistakes
+when creating their accounts, or group accounts might need to be changed,
+or for several other reasons.
```

Comment:
I would replace this paragraph with just "Sometimes it is desirable to rename an OCF account."